frontend/packages/console-shared/src/components/dashboard/launcher-card/LauncherBody.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react';
+
+import './launcher-card.scss';
+
+const LauncherBody: React.FC = ({ children }) => (
+  <div className="co-dashboard-card__body--top-margin co-launcher-card__body">{children}</div>
```

Comment:
`co-dashboard-card__body--top-margin` is this needed? It feels a bit odd to add a class this specific to a property. 
Does that card not come with the proper spacing?

Reply:
pf4 card body does not have a top margin. Some of our cards dont have it either (like activity card) so any card that needs it has to define the class name.

Reply:
ok sounds good. I think there is talk about revising the cards to improve this in the future so we *may* not need this modifier in the future.